src/jloptions.c

```diff
@@ -43,7 +86,13 @@ static const char opts[]  =
     // code generation options
     " --compile={yes|no|all|min}Enable or disable JIT compiler, or request exhaustive compilation\n"
     " -C, --cpu-target <target> Limit usage of cpu features up to <target>\n"
-    " -O, --optimize={0,1,2,3}  Set the optimization level (default 2 if unspecified or 3 if specified as -O)\n"
+    " -O, --optimize={0,1,2,3}  Set the optimization level (default is 2 if unspecified or 3 if specified as -O)\n"
+    " -g, -g <level>            Enable / Set the level of debug info generation (default is "
+#ifdef JL_DEBUG_BUILD
+        "2, the same as -g)\n"
```

Comment:
"default is 2, the same as -g" doesn't make sense - this *is* the help string for "-g" !